<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWidgetDialog</class>
 <widget class="QDialog" name="ListWidgetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List widget Dialog</string>
  </property>
  <widget class="QListWidget" name="leftList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="rightList">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMoveLeft">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMoveRight">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
